adserver/mixins.py

```diff
@@ -43,3 +47,29 @@ def test_func(self):
             self.request.user.is_staff
             or publisher in self.request.user.publishers.all()
         )
+
+
+class EstimatedCountPaginator(Paginator):
+
+    """
+    Paginator that gives only an estimated count based on DB statistics..
+
+    This only works for PostgreSQL. Other database engines always return the full count.
+    """
+
+    @cached_property
+    def count(self):
+        if "postgresql" not in settings.DATABASES["default"]["ENGINE"]:
```

Comment:
Could turn it off when `DEBUG` is `True`, but doesn't seem too necessary.

Reply:
That's a good idea!